File: JavaScript/ifStatements/movieRating.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})
    
function movieRatings(age) {
    let ratings = [];
    if (age >= 17) {
        ratings.push('R');
    }
    if (age >= 13) {
        ratings.push('PG-13');
    }
    ratings.push('G'); 
    return ratings;
}
rl.question("Enter your age please. ", (input) => {
    const age = parseInt(input);

    if (isNaN(age) || age < 0) {
        console.log("Invalid age entered.");
    } else {
        const ratings = movieRatings(age);
        console.log(`You are allowed to watch movies rated: ${ratings.join(', ')}`);
    }
    rl.close();
}); 

File: ifStat/src/movieRating.java
import java.util.Scanner;

public class movieRating {
    public static void main(String[] args){

       Scanner sc = new Scanner(System.in);
       System.out.println("Please enter your age!");
       int age = sc.nextInt();

       if(age >= 18){
           System.out.println("You are old enough to watch R rated movies!");
       } else if (age >= 13 && age <= 17){
           System.out.println("You are allowed to watch PG-13, or G rated movies");
       } else if (age < 12){
           System.out.println("You are  allowed to watch G rated movies");
       }
    }
}
